packages/vite/src/node/config.ts

```diff
@@ -145,8 +145,9 @@ export interface UserConfig {
   preview?: PreviewOptions
   /**
    * Dep optimization options
+   * false disables optimization completely (experimental)
```

Comment:
```suggestion
   * Dep optimization options
   *
   * false disables optimization completely (experimental)
```

I think we need to add an empty line here, or a full-stop, so that intellisense displays the docs correctly. An empty line would create a line break. Currently intellisense would concat these into one sentence.

<img width="483" alt="Screenshot 2022-04-12 at 1 36 29 PM" src="https://user-images.githubusercontent.com/34116392/162888205-9b51fda1-4c8b-40af-83f8-23716661d0bf.png">